utils/generatePairs.ts

```diff
@@ -50,4 +50,31 @@ function shuffle(array: string[]): string[] {
   return array;
 }
 
-export default generatePairs;
\ No newline at end of file
+export const shiftByOne = async (): Promise<string[][]>  => {
```

Comment:
would `shiftByOne` be used inside the `generatePairs` function above?

Reply:
I think it would just be called instead after the first call no?

Reply:
ohi see